Documentation/Internal/ContainerLogV2-Linux.xml

```diff
@@ -25,6 +25,8 @@
       <Column name="Computer" type="str" mdstype="mt:wstr" />
       <Column name="ContainerName" type="str" mdstype="mt:wstr" />
```

Comment:
1. bump the version 1.0 to 1.1
2.  lets validate these changes works for Geneva path.

Reply:
changed